docs/json.md

```diff
@@ -197,6 +197,13 @@ Remark: If an enum is used, it should be checked if one of the following points
 
 In case additional values for the enum should be added compatibly later, a default value must always be specified (see [Format Versions and Compatibility](#format-versions-and-compatibility)). If systems don't support the new enumeration value (e.g., in lower releases), the value will be changed to the default value by the file format implementations.
 
+### Regular Expressions
+
+In order to express constraints to the values of a field, it is possible to pass [regualr expressions](https://json-schema.org/understanding-json-schema/reference/regular_expressions) enclosed in single quotes:
```

Comment:
```suggestion
In order to express constraints to the values of a field, it is possible to pass [regular expressions](https://json-schema.org/understanding-json-schema/reference/regular_expressions) enclosed in single quotes:
```

Reply:
Thanks for the hint. I moved the section in between `Length Specification` and `Mutliple Of`. I simply oversaw that it was in the middle of the `Enum Values`.